### DISCUSSION TITLE
libiomp5md.dll error

### DISCUSSION THREAD

**Discussion by jakubMitura14** [Project-MONAI/MONAI#3504] [Q&A] [answered]
Hello when I import monai I get libiomp5md.dll error - multiple copien of OpenMP have been linked into the program ... - I suppose some other process on Pc is using OpenMp (thy script with monai for now is just install and import statement) How to establish what it can be?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @jakubMitura14 ,

Thanks for your interest and feedback here.
Could you please help share your test program to reproduce the issue?

Thanks in advance.

**Reply by jakubMitura14** [reply]
I tested that the problem is not related to monai (I was colling monai from Julia lang and pyCall when I execute pure python script it works) but thanks !

**Reply by Nic-Ma** [reply]
No problem, please feel free to submit a ticket if you have any other questions.

Thanks.
